final-project/styles/main.scss

```diff
@@ -1,13 +1,820 @@
+
+// ------------------------------------------
+//                 STATS
+
+.stats{
+  display: flex;
+  font-family: 'Quicksand', sans-serif;
+  justify-content: center;
+}
+
+.stats > div {
+  height: 342px;
+  width: 50%;
+}
+
+.stats__productivity {
+  background-color: #6888FF;
+  text-align: right;
+}
+
+.stats__productivity__content {
+  padding: 66px 90px 66px 30px;
+}
+
+.stats__productivity__content > p {
+  color: #FFF;
+  font-size: 18px;
+  padding-top: 24px;
+}
+
+.stats__productivity > img {
+  width: 65px;
+}
+
+.stats__users {
+  background-color: #E9EFFF;
+}
+
+.stats__users__content {
+  padding: 66px 30px 66px 90px;
+}
+
+.stats__users__content > p {
+  color: #5E80FF;
+  font-size: 18px;
+  padding-top: 24px;
+}
+.stats__users > img {
+  width: 71px;
+}
+
+.stats .number {
+  font-size: 48px;
+}
+
+//               ! STATS
+// ------------------------------------------
+
+// ------------------------------------------
+//                 BLOG
+
+.blog {
+  background-color: #F5F8FF;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 864px;
+  position: relative;
+}
+
+.blog__content {
+  margin: 0 auto;
+  max-width: 1440px;
+  padding: 94px 0 0 0;
+  text-align: center;
+  width: 100%;
+}
+
+.blog__content h1 {
+  padding: 10px;
+}
+
+.blog .carousel__icon {
+  align-self: center;
+  display: none;
+  padding-bottom: 100px;
+}
+
+.blog__title {
+  color: #3D4F9F;
+  font-family: 'Quicksand', sans-serif;
+  font-size: 30px;
+  font-weight: 500;
+  padding: 10px 0 46px;
+}
+
+.blog__carousel {
+  align-items: center;
+  display: none;
+  justify-content: space-evenly;
+  padding: 46px 0;
+}
+
+.carousel__box {
+  align-items: center;
+  background-color: #FFF;
+  display: flex;
+  max-width: 958px;
+  min-height: 394px;
+  text-align: left;
+  width: 100%;
+}
+
+.carousel__box > div {
+  float: left;
+  padding: 35px;
 }
+
+.blog_title {
+  color: #3D4F9F;
+  font-family: 'Quicksand', sans-serif;
+  font-size: 36px;
+}
+
+.blog_description {
+  font-size: 14px;
+  line-height: 1.6;
+  margin: 32px 0;
+}
+
+.carousel__box > div.blog__text {
+  padding-bottom: 24px;
+}
+
+.blog__text > a.blog_url {
+  background-color: #D8E3FE;
+  color: #3A79FF;
+}
+
+.blog__text > a{
+  background-color: #FFF;
+  border-radius: 2px;
+  color: #A6B6DA;
+  font-family: 'Quicksand', sans-serif;
+  font-size: 13px;
+  font-weight: 700;
+  margin: 4px;
+  padding: 13px 16px;
+  text-decoration: none;
+}
+
+.blog_img_tablet {
+  display: none;
+}
+.blog_img_mobile_2x {
+  display: none;
+}
+.blog_img_mobile {
+  display: none;
+}
+
+.blog__button {
+  background-color: transparent;
+  border-radius: 50%;
+  border: none;
+  color: #A8B4E5;
+  cursor:pointer;
+  height: 20px;
+  outline:0;
+  width: 20px;
+}
+
+.loading_blog > p{
+  font-size: 30px;
+}
+
+.loading_blog > img {
+  margin-top: 30px;
+  padding: 50px;
+  width: 100px;
+}
+
+//               ! BLOG
```

Comment:
This will get large really soon, just like js files you can import Sass files and separate each section in a file